server/src/main/java/io/whitefox/api/deltasharing/server/DeltaSharesApiImpl.java

```diff
@@ -86,7 +119,7 @@ public CompletionStage<Response> listSchemas(String share, Integer maxResults, S
                 .map(t -> new ListSchemasResponse().nextPageToken(t).items(ct.getContent()))
                 .orElse(new ListSchemasResponse().items(ct.getContent())))
             .build()))
-        .exceptionallyAsync(exceptionToResponse);
+        .exceptionally(exceptionToResponse);
```

Comment:
Why did you change all of these?

Reply:
because I changed the excpetionToResponse method. But later I rolled back